apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -49,6 +49,29 @@ public function register_scripts() {
 			filemtime( plugin_dir_path( __FILE__ ) . 'dist/starter-page-templates.js' ),
 			true
 		);
+		wp_register_script(
+			'starter-page-templates-error',
+			null,
+			array(),
+			'1.O',
```

Comment:
I intended to use `false` here but phpcs didn't like that so I just picked a `1.0` version string. Doesn't change anything as only the inline script makes it into the final HTML page.

Reply:
Do we need that even? Is there not a Core script we can latch onto?

Reply:
I'm not sure. Would that be reliable? This way we are in full control and our script won't possibly be filtered out which might happen to core ones. 

When I researched how to enqueue an inline script, it came to two solutions:
- file-less script enqueue + `wp_add_inline_script`
- or hooking into `wp_footer` and `echo` the whole script markup

The first one felt nicer to me.

Reply:
Could we maybe not register the script first? I don't know, it feels like a lot of hoops to jump through for debug info.

Reply:
It needs to be registered I think (the inline script was ignored otherwise) but we can do that only once really needed. I pushed an update.

Reply:
WordPress is pretty picky about registering inline scripts. 

I do feel more comfortable with us being in full control even if it is a little verbose.